src/EA.Weee.Web.Tests.Unit/Areas/Admin/Controllers/ObligationsControllerTests.cs

```diff
@@ -248,8 +259,16 @@ public void UploadObligationsPost_IsDecoratedWith_HttpPostAttribute()
         public void UploadObligationsPost_IsDecoratedWith_ValidateAntiForgeryTokenAttribute()
         {
             typeof(ObligationsController).GetMethod("UploadObligations", BindingFlags.Public | BindingFlags.Instance, null, CallingConventions.Any, new Type[] { typeof(UploadObligationsViewModel) }, null)
-            .Should()
-            .BeDecoratedWith<ValidateAntiForgeryTokenAttribute>();
+                                         .Should()
+                                         .BeDecoratedWith<ValidateAntiForgeryTokenAttribute>();
+        }
+
+        [Fact]
+        public void UploadObligationsPost_ShouldHaveAuthorizeInternalClaimsAttribute()
```

Comment:
@sreedhar-civica Think we may just need a test for the UploadObligations GET method that is has the InternalAdmin claim attribute.

Reply:
@JohnPadgettCivica  Yes, John. I have added that test now. Please check.